bin/migrate-oats-data/srw/sql/srw_staff_journal.sql

```diff
@@ -0,0 +1,3 @@
+SELECT osj.journal_date, osj.journal_text, osj.revision_count, osj.staff_journal_entry_id, an."uuid" 
+FROM oats.oats_staff_journal_entries osj
+JOIN alcs.notification an ON an.file_number = osj.alr_application_id::TEXT 
```

Comment:
could you add SRW filter here since it is possible that later we will have different types on notification table

Reply:
Is there a reason we not want those to propagate to staff journals? I would expect every notification with a staff journal associated with it we would want in this table

Reply:
The short answer is data integrity. Long answer: the notification table could store not only SRW but any other type. It means that for subsequent ETLs you need to keep in mind that anything other than records linked to SRW could be outdated in ALCS DB or even not relevant to the ALCS version of the feature. As a result, you could end up with "somewhat correct" data which makes it unreliable

Reply:
Updated with SRW filter